src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  static alive = [];
+  static deleteIndex = Animal.alive.findIndex((animal) => animal.health <= 0);
+  constructor(health, name) {
```

Comment:
The constructor should not have `health` as a parameter since it is always initialized to `100` as per the task requirements. Remove the `health` parameter to avoid confusion.